docs/source/changelog.rst

```diff
@@ -19,12 +19,16 @@ Breaking Changes
 * The ``backend`` argument to the |Results| middleware is now required.
   Previously, not supplying this argument would result in a non-functional |Results| middleware.
   (`#728`_, `@LincolnPuzey`_)
+* The long-deprecated ``requeue_{deadline,interval}`` parameters of |RedisBroker|,
+  as well as the obsolete heartbeat compat code, have been removed. (`#771`_, `@mikeroll`_)
```

Comment:
Not _that_ breaking I suspect but technically we're still changing the interface.